frontend/packages/dev-console/src/utils/__tests__/shared-submit-utils.spec.ts

```diff
@@ -0,0 +1,42 @@
+import * as _ from 'lodash';
+import { mockFormData } from '../../components/import/__mocks__/import-validation-mock';
+import { mockDeployImageFormData } from '../../components/import/__mocks__/deployImage-validation-mock';
+import { createService, createRoute } from '../shared-submit-utils';
+import { GitImportFormData, DeployImageFormData } from '../../components/import/import-types';
+
+describe('Shared submit utils', () => {
```

Comment:
Probably worth creating some snapshot tests here so that the entire output can be tested fully.

Reply:
Done.